src/store/catalog/selectors.ts

```diff
@@ -0,0 +1,16 @@
+import  type { RootStore } from "@store/store";
+import type { State } from "./slice";
+
+const getCatalog = (store: RootStore): State => store.catalog;
```

Comment:
обычно селектор слайса имеет постфикс Slice, чтобы было сразу понятно что это корневой селектор
```suggestion
const getCatalogSlice = (store: RootStore): State => store.catalog;
```